modules/scheduler/gke-cluster/variables.tf

```diff
@@ -262,7 +262,7 @@ variable "authenticator_security_group" {
 variable "enable_dataplane_v2" {
   description = "Enables [Dataplane v2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2). This setting is immutable on clusters."
```

Comment:
Add null meaning to description. "If null, will default to false unless using multi-networking, in which case it will default to true"